cairo/ethereum/cancun/vm.cairo

```diff
@@ -608,3 +609,28 @@ namespace EvmImpl {
         return ();
     }
 }
+
+namespace EnvImpl {
```

Comment:
Problem with rebase ?
There is already a namespace EnvImpl

Reply:
EvmImpl, not EnvImpl, is it?